pints/toy/_twisted_gaussian_banana.py

```diff
@@ -68,6 +68,16 @@ def __call__(self, x):
         y[1] += self._b * ((x[0] ** 2) - self._V)
         return self._phi.logpdf(y)
 
+    def detransform(self, samples):
```

Comment:
Why not call it "untwist" since we don't mention transforming anywhere (and untwisting seems more evocative) :).